iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/MergedLine.java

```diff
@@ -95,6 +97,26 @@ private void mergeProperties(DanglingLine dl1, DanglingLine dl2) {
         });
     }
 
+    void computeAndSetP0() {
+        double p1 = dl1.getTerminal().getP();
+        double p2 = dl2.getTerminal().getP();
+        if (!Double.isNaN(p1) && !Double.isNaN(p2)) {
+            double p0 = (p1 + p2) / 2.0d;
+            dl1.setP0(p0);
+            dl2.setP0(-p0); // sign depends of side
```

Comment:
No, it depends on the transit flow: `P1 ---->-----DL1.P0 ---->----- DL2.P0 ---->---- P2`

The sign of DL1.P0 is the same as P2, and respectively the sign of DL2.P0 is the same than P1